ios/Podfile

```diff
@@ -30,7 +30,9 @@ target 'app' do
     #
     # Note that if you have use_frameworks! enabled, Flipper will not work and
     # you should disable the next line.
-    :flipper_configuration => FlipperConfiguration.enabled,
+    :flipper_configuration => ENV['CI'] ?
+        FlipperConfiguration.disabled :
+        FlipperConfiguration.enabled,
```

Comment:
It could be also configured to enabled on staging app? as tests? But for that we need to enable  `FB_SONARKIT_ENABLED`